src/main/resources/templates/db_scripts/schema.json

```diff
@@ -4,11 +4,16 @@
       "run": "after",
       "snippet": "CREATE OR REPLACE VIEW receiving_history_view AS SELECT pieces.id AS id, json_build_object('id', pieces.jsonb->>'id', 'caption', pieces.jsonb->>'caption', 'comment', pieces.jsonb->>'comment', 'dateOrdered', purchase_order.jsonb->>'dateOrdered', 'checkin', po_line.jsonb->>'checkinItems', 'instanceId', po_line.jsonb->>'instanceId','itemId', pieces.jsonb->>'itemId', 'locationId', pieces.jsonb->>'locationId', 'pieceFormat', pieces.jsonb->>'format', 'poLineId', pieces.jsonb->>'poLineId', 'poLineNumber', po_line.jsonb->>'poLineNumber', 'purchaseOrderId', po_line.jsonb->>'purchaseOrderId', 'poLineReceiptStatus', po_line.jsonb->>'receiptStatus','receivedDate', pieces.jsonb->>'receivedDate', 'title', po_line.jsonb->>'title','receivingNote', po_line.jsonb->'details'->>'receivingNote', 'receivingStatus', pieces.jsonb->>'receivingStatus', 'supplement', pieces.jsonb->>'supplement') AS jsonb FROM pieces LEFT OUTER JOIN po_line ON pieces.jsonb->>'poLineId' = po_line.jsonb->>'id' LEFT OUTER JOIN purchase_order ON po_line.jsonb->>'purchaseOrderId' = purchase_order.jsonb->>'id';",
       "fromModuleVersion": "1.0"
-    }
-  ,{
+    },
+    {
       "run": "after",
       "snippet": "CREATE OR REPLACE VIEW orders_view AS SELECT purchase_order.id AS id, purchase_order.jsonb AS jsonb, po_line.jsonb || COALESCE(purchase_order.jsonb, '{}'::jsonb) AS metadata FROM po_line LEFT JOIN purchase_order ON (po_line.jsonb -> 'purchaseOrderId'::text) = (purchase_order.jsonb -> 'id'::text);",
       "fromModuleVersion": "1.0"
+    },
+    {
+      "run": "after",
+      "snippet": "CREATE OR REPLACE VIEW order_lines_view AS SELECT po_line.id AS id, po_line.jsonb AS jsonb, po_line.jsonb || COALESCE(purchase_order.jsonb, '{}'::jsonb) AS metadata FROM po_line LEFT JOIN purchase_order ON (po_line.jsonb -> 'purchaseOrderId'::text) = (purchase_order.jsonb -> 'id'::text);",
```

Comment:
`po_line.jsonb || COALESCE(purchase_order.jsonb, '{}'::jsonb) AS metadata` will join all properties from both tables. However there is one property `id` which presents in both. I believe that the `id` will be chosen from purchase_order.jsonb.
In case CQL `query` contains `...id=123 or ...` could you please check what SQL will be generated for such query?

Reply:
That is true common key fields will be replaced with the value from the right hand operand. hence metadata.id=purchase_order.id. 
I think it makes sense to change it to be metadata.id=po_line.id